test/Visualization_Tests/src/Lazy_Text_Spec.enso

```diff
@@ -0,0 +1,21 @@
+from Standard.Base import all
+
+import Standard.Examples
+
+import Standard.Visualization.Text as TextVis
+import Standard.Visualization.Preprocessor as Preprocessor
+
+from Standard.Test import Test
+
+sample_text_single_line = "ABCDEFGHIJKLMNOPQRS"
+sample_text_multi_line = """
+    ABCDEFGHIJKLMNOPQRS
+    1234567890
+
+spec = Test.group "By coordinates" <|
+    Test.specify "Get correct chunks" <|
+        (Preprocessor.lazy_preprocessor sample_text_multi_line [0,0] [1,1] 5).should_equal '{"chunks":"[[[0, 0], \\"ABCDE\\"]]","lines":2,"max_line_length":19}'
```

Comment:
Could consider using `Pair` instead of `Vector` for the API: `Pair.new 0 0` instead of `[0, 0]` but as an internal function am happy with Vector or an Array if easier.